pylmod/tests/test_gradebook.py

```diff
@@ -17,6 +18,7 @@
 from pylmod.tests.common import BaseTest
 
 
+@ddt
```

Comment:
I know, but I've seen `ddt` being used in edx and the class was not decorated, only the test methods. what happens if you remove this?